iso:std:iso:34000:ed-1:v1:en.xml

```diff
@@ -0,0 +1,1556 @@
+      <p>ISO documents relating to date and time concepts have been available since 1971.</p>
+      <p>This document presents terms and definitions for selected concepts relevant to date and time concepts and of their representation.</p>
+      <p>Specifically, the terminology presented in this document:</p>
+      <list list-type="dash">
+        <list-item>
+          <label>&#x2014;</label>
+          <p>serves as a sound basis in the understanding of date and time;</p>
+        </list-item>
+        <list-item>
+          <label>&#x2014;</label>
+          <p>guides new developments in the field by underpinning mutual understanding;</p>
+        </list-item>
+        <list-item>
+          <label>&#x2014;</label>
+          <p>serves as a quick and handy reference for those newly inaugurated to this field.</p>
+        </list-item>
+      </list>
+      <p>In this document, the decimal sign is a comma on the line, and each group of three digits are separated by a small space from the preceding digits, counting from the decimal sign, in accordance with the ISO/IEC Directives, Part 2.</p>
+      <p>However, Resolution 10 of the 22nd General Conference on Weights and Measures (Conf&#xE9;rence G&#xE9;n&#xE9;rale des Poids et Mesures, CGPM) in 2003 stated that:</p>
+      <p>&#x201C;The decimal marker shall be either a point on the line or a comma on the line.&#x201D;</p>
+      <p>And reaffirmed the following resolution from Resolution 7 of the 9th CGPM, 1948:</p>
+      <p>&#x201C;Numbers may be divided in groups of three in order to facilitate reading.&#x201D;</p>
+      <p>In practice, the choice between these alternatives depends on customary use in the language concerned. In the technical areas of date and time, it is customary for the decimal point always to be used, and that numbers are not grouped, for all languages.</p>
+    </sec>
+  </front>
+  <body>
+    <sec id="sec_1">
+      <label>1</label>
+      <title>1   Scope</title>
+      <p>This document defines terms related to date and time, from fundamental concepts to those of their usage and representation.</p>
+    </sec>
+    <sec id="sec_2">
+      <label>2</label>
+      <title>2   Normative references</title>
+      <p>There are no normative references in this document.</p>
+    </sec>
+    <sec id="sec_3">
+      <label>3</label>
+      <title>3   Terms related to general concepts</title>
+      <p>ISO and IEC maintain terminology databases for use in standardization at the following addresses:</p>
+      <list list-type="dash">
+        <list-item>
+          <label>&#x2014;</label>
+          <p>ISO Online browsing platform: available at</p>
+        </list-item>
+        <list-item>
+          <label>&#x2014;</label>
+          <p>IEC Electropedia: available at</p>
+        </list-item>
+      </list>
+      <term-sec id="sec_3.1">
+        <label>3.1</label>
+        <tbx:termEntry id="term_3.1">
+          <tbx:langSet xml:lang="en">
+            <tbx:tig>
```

Comment:
Do we need to generate an id for this element?

Reply:
Refer to the full 34000 XML?

Reply:
Sorry, I lacked precision here.

The id format of the `tbx:tig` tag here is `term-date-1_preferred`

To generate it, here's the method for each:
- prefix `term-`
- term name with hyphens instead of spaces
- suffix `-1_preferred`

Does this work for you?

Reply:
Yup. The phrase "preferred" designates the "type" of the term, it could be "preferred", "admitted" or "deprecated".

Reply:
When we look in the html received for a term of type `admitted`, for example `timescale` here
https://github.com/metanorma/obp-access/pull/12/files#diff-4b8cf304e0cc89bd020219677f15aa46c7ded96d781c67aaa61643d49374eea3R319

There's no information on whether to use “preferred”, “admitted” or “deprecated”.

And when you look in the `iso_std_iso_34000_ed-1_v1_en.xml` file, the method for generating the `tig id` is not the same. The ids seem to be like this:
```xml
<tbx:tig id=“term_3.2-1”>
```

What is the correct format for generating the id?

Reply:
> There's no information on whether to use “preferred”, “admitted” or “deprecated”.

See https://github.com/metanorma/obp-access/pull/12#discussion_r1848564908

> What is the correct format for generating the id?

Let's follow `iso_std_iso_34000_ed-1_v1_en.xml`. Sometimes things are not always consistent... because this file is somewhat human made...